content/docs/08-troubleshooting/01-migration-guide/01-migration-guide-4-0.mdx

```diff
@@ -250,6 +250,11 @@ LangChainAdapter.toDataStream(stream);
 
 ### `streamText` returns immediately
 
+<Note type="warning">
+  There is no codemod available for this change. Please review and update your
+  code manually.
+</Note>
```

Comment:
would be good to have this one and the one for streamObject since they will be super common, if time permits and they are easy (just the removal of the await, no promise handling)

Reply:
Done in #3724